src/rememberizer/README.md

```diff
@@ -0,0 +1,173 @@
+# Rememberizer MCP server
+
+## Overview
+
+A Model Context Protocol server for interacting with Rememberizer's document and knowledge management API. This server enables Large Language Models to search, retrieve, and manage documents and integrations through Rememberizer.
+
+Please note that mcp-server-rememberizer is currently in development and the functionality may be subject to change.
+
+## Components
+
+### Resources
+
+The server provides access to two types of resources:
+- Documents (`rememberizer://document/{id}`)
+- Slack discussions (`rememberizer://slack/{id}`)
+
+### Tools
+
+1. `rememberizer_search`
+   - Search for documents by semantic similarity
+   - Input:
+     - `q` (string): Up to a 400-word sentence to find semantically similar chunks of knowledge
+     - `n` (integer, optional): Number of similar documents to return
+   - Returns: Search results as text output
+
+2. `rememberizer_list_integrations`
+   - List available data source integrations
+   - Input: None required
+   - Returns: List of available integrations
+
+3. `rememberizer_account_information`
+   - Get account information
+   - Input: None required
+   - Returns: Account information details
+
+## Installation
+
+### Using uv (recommended)
+
+When using [`uv`](https://docs.astral.sh/uv/), no specific installation is needed. Use [`uvx`](https://docs.astral.sh/uv/guides/tools/) to directly run *mcp-server-rememberizer*.
+
+### Using PIP
+
+Install `mcp-server-rememberizer` via pip:
+
+`pip install mcp-server-rememberizer`
+
+After installation, run it as a script:
+
+`python -m mcp_server_rememberizer`
+
+## Configuration
+
+### Environment Variables
+
+The following environment variables are required:
+- `REMEMBERIZER_API_TOKEN`: Your Rememberizer API token
```

Comment:
Please add a link to our Rememberizer docs to instruct them on how to get our API token. Thanks.